primeorder/src/projective.rs

```diff
@@ -405,7 +405,7 @@ where
         out[i] = C::AffinePoint::conditional_select(
             &points[i].to_affine_internal(zs_inverses.as_ref()[i]),
             &C::AffinePoint::IDENTITY,
-            points[i].z.ct_eq(&C::FieldElement::ZERO),
+            points[i].z.normalizes_to_zero(),
```

Comment:
Only `k256` uses lazy normalization, so this isn't needed here

Reply:
Fixed